lib/_tls_wrap.js

```diff
@@ -437,49 +478,56 @@ TLSSocket.prototype._finishInit = function() {
 };
 
 TLSSocket.prototype._start = function() {
```

Comment:
Is it possible that the `once` below could set multiple listeners for `connect`, causing `_start` to be called >1?